src/test/java/com/mojang/brigadier/CommandDispatcherTest.java

```diff
@@ -70,6 +81,14 @@ public void testCreateAndExecuteOffsetCommand() throws Exception {
         verify(command).run(any(CommandContext.class));
     }
 
+    @Test
+    public void testCreateAndExecuteCaseInsensitiveOffsetCommand() throws Exception {
+        subject.register(literal("Foo").executes(command).caseInsensitive(true));
+
+        assertThat(subject.execute(inputWithOffset("/foo", 1), source), is(42));
+        verify(command).run(any(CommandContext.class));
+    }
+
```

Comment:
Additional tests for using ignoreCase as a command argument should be added. Currently this only these tests only test the base node.